src/components/vm/nics/vmNicsCard.tsx

```diff
@@ -499,7 +499,7 @@ export class VmNetworkTab extends React.Component<VmNetworkTabProps, VmNetworkTa
                                     </React.Fragment>
                                 ))}
                             </DescriptionListGroup>}
-                            {inet6Ips.length && <DescriptionListGroup>
+                            {inet6Ips.length > 0 && <DescriptionListGroup>
```

Comment:
On the other hand, would it hurt us to drop the conditional entirely and always have the `<DescriptionListGroup>`, possibly empty?